winui/Masked-TextBox/Culture.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Culture in WinUI MaskedTextBox
 
-The MaskedTextBox allows you to set any culture support for currency symbols, date separators, time separators, group separators, and decimal separators by using the below special symbols.
```

Comment:
@Vignesh-SF3580 Split the word MaskedTextBox in all pages.

Reply:
Splited the word MaskedTextBox in all pages.